cpp/log/cert.cc

```diff
@@ -1120,30 +1122,33 @@ Cert::Status TbsCertificate::DeleteExtension(int extension_nid) {
 
   // ExtensionIndex returns the first matching index - if the extension
   // occurs more than once, just give up.
-  int ignored;
-  status = ExtensionIndex(extension_nid, &ignored);
-  if (status == Cert::TRUE) {
+  status = ExtensionIndex(extension_nid);
```

Comment:
Could use a separate (`const`) `ignored` (or `ignored_index`), instead of reusing the one above?